### DISCUSSION TITLE
Sliders created in osu!lazer editor are distorted after opening in osu!stable editor

### DISCUSSION THREAD

**Discussion by domintes** [ppy/osu#23843] [Q&A]
### Type

Game behaviour

### Bug description

When map is created in osu!lazer editor, some sliders are distorted when map is opened in osu!stable editor.
Maybe is issue with slider curve type, I noticed this bug especially on sliders with 'Bezier', 'Perfect Curve' and 'Catmull' curve type.
I guess osu!stable editor can't read correctly created in osu!lazer editor sliders with some specific curve types, so probably that's reason of this bug

Below in the image are examples of this bug:

### Screenshots or videos

![sliderbugs](https://github.com/ppy/osu/assets/30428748/35891979-3fe5-4b1f-b12a-1946093419ca)

### Version

2023.605.0-lazer

### Logs

 ឵឵
```[tasklist]
### Tasks
```

**Comment by peppy** [comment]
This is being tracked already, and a known issue. You can still make sliders work correctly if you use the same curve types throughout the slider. The issue occurs when you use multiple curve types in a single slider.

By the way, you can make sliders with a lot less effort now. Here's an example of how to make a heart:

![osu! 2023-06-09 at 09 52 27](https://github.com/ppy/osu/assets/191335/900e856d-342c-4bba-8b72-0ad99bf6a669)

**Comment by Axilotl17** [comment]
https://github.com/JPK314/LazerToStable or https://github.com/Speykious/oSUS

i use the first one; download the zip, go into directory, and run `python convert <file path>` or `python3 convert <file path>`

**Reply by Axilotl17** [reply]
not a permanent solution but temporary solution to convert the maps
